unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -220,6 +221,7 @@ internal IEnumerator LoadNFTAssetCoroutine(NFTInfo nftInfo)
                 ShowErrorFeedback(true);
                 OnLoadingAssetFail?.Invoke();
                 FinishLoading(false);
+                throw exc;
```

Comment:
This throwing may f*ck up the execution in the browser build, I'm not sure, but I guess if we see it happen we may revisit this

Reply:
my bad, I added this since I was not getting any logs when the NFT's failed